cron/src/main/scala/com/criteo/cuttle/cron/CronModel.scala

```diff
@@ -0,0 +1,56 @@
+package com.criteo.cuttle.cron
+
+import java.time.temporal.ChronoUnit
+import java.time.{Duration, Instant, ZoneOffset}
+
+import com.criteo.cuttle.{Job, Scheduling, SchedulingContext, Workload}
+import cron4s.{Cron, _}
+import cron4s.lib.javatime._
+
+import scala.concurrent.duration._
+
+private[cron] case class ScheduledAt(instant: Instant, delay: FiniteDuration)
+
+/** A [[CronContext]] is passed to [[com.criteo.cuttle.Execution executions]] initiated by
+  * the [[CronScheduler]].
+  */
+private[cron] case class CronContext(instant: Instant)(retryNum: Int) extends SchedulingContext {
+  val retry: Int = retryNum
+
+  def compareTo(other: SchedulingContext): Int = other match {
+    case CronContext(otherInstant) =>
+      instant.compareTo(otherInstant)
+  }
+}
+
+/** Configure a [[com.criteo.cuttle.Job job]] as a [[CronScheduling]] job.
+  *
+  * @param cronExpression Cron expression to be parsed by https://github.com/alonsodomin/cron4s.
+  *                       See the link above for more details.
+  */
+case class CronScheduling(cronExpression: String) extends Scheduling {
```

Comment:
Se we are missing `maxRetry` in the scheduling configuration

Reply:
added it